packages/components/spinner/src/Spinner.tsx

```diff
@@ -0,0 +1,357 @@
+import { cx, css, keyframes } from 'emotion';
+import React, { forwardRef } from 'react';
+import tokens from '@contentful/f36-tokens';
+import { Box } from '@contentful/f36-core';
+import type {
+  BoxInternalProps,
+  ComponentVariant,
+  PickUnion,
+  PolymorphicComponent,
+  PolymorphicComponentProps,
+  PolymorphicComponentWithRef,
+  Simplify,
+} from '@contentful/f36-core';
+
+const DEFAULT_TAG = 'div';
+
+export type SpinnerVariant = Simplify<
+  PickUnion<ComponentVariant, 'primary'> | 'default' | 'white'
```

Comment:
TIL 🦺

Reply:
This uses our own types so it won't work everywhere out of the box 🙂